public/app/plugins/datasource/loki/languageUtils.ts

```diff
@@ -111,3 +113,56 @@ export function getLabelTypeFromFrame(labelKey: string, frame?: DataFrame, index
       return null;
   }
 }
+
+export const mapOpToAbstractOp: Record<AbstractLabelOperator, string> = {
+  [AbstractLabelOperator.Equal]: '=',
+  [AbstractLabelOperator.NotEqual]: '!=',
+  [AbstractLabelOperator.EqualRegEx]: '=~',
+  [AbstractLabelOperator.NotEqualRegEx]: '!~',
+};
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
This eslint disable doesn't seem necessary. I removed it and I saw no complaint.

Reply:
Without it, it will be added to betterer, which we don't want, because we want to use this on purpose.